From a5f7ffbe3bb4cbb2a24b7e37d0119231f4cbb984 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 26 Feb 2020 19:55:56 +0200
Subject: [PATCH 111/186] rework Arbutus font

	modified:   data/fonts/fonts.xml
	modified:   packages/overlays/FontArbutusOverlay/AndroidManifest.xml
	modified:   packages/overlays/FontArbutusOverlay/res/values/config.xml
---
 data/fonts/fonts.xml                                       | 1 +
 packages/overlays/FontArbutusOverlay/AndroidManifest.xml   | 2 +-
 packages/overlays/FontArbutusOverlay/res/values/config.xml | 4 ++--
 3 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/data/fonts/fonts.xml b/data/fonts/fonts.xml
index 08d19c7f505..1b07b374ce7 100644
--- a/data/fonts/fonts.xml
+++ b/data/fonts/fonts.xml
@@ -116,6 +116,7 @@
         <font weight="400" style="normal">ArbutusSlab-Regular.ttf</font>
     </family>
     <alias name="arbutus-slab-medium" to="arbutus-slab" weight="500" />
+    <alias name="arbutus-slab-bold" to="arbutus-slab" weight="700" />
 
     <family name="arvo">
         <font weight="400" style="normal">Arvo-Regular.ttf</font>
diff --git a/packages/overlays/FontArbutusOverlay/AndroidManifest.xml b/packages/overlays/FontArbutusOverlay/AndroidManifest.xml
index 7e362884243..f1ed244df43 100644
--- a/packages/overlays/FontArbutusOverlay/AndroidManifest.xml
+++ b/packages/overlays/FontArbutusOverlay/AndroidManifest.xml
@@ -26,6 +26,6 @@
     <application android:label="@string/font_arbutus_overlay" android:hasCode="false">
         <meta-data
             android:name="android.theme.customization.REQUIRED_SYSTEM_FONTS"
-            android:value="arbutus-slab,arbutus-slab-medium" />
+            android:value="arbutus-slab,arbutus-slab-bold" />
     </application>
 </manifest>
diff --git a/packages/overlays/FontArbutusOverlay/res/values/config.xml b/packages/overlays/FontArbutusOverlay/res/values/config.xml
index 3b5224e8ae8..253e4a12c0e 100644
--- a/packages/overlays/FontArbutusOverlay/res/values/config.xml
+++ b/packages/overlays/FontArbutusOverlay/res/values/config.xml
@@ -19,9 +19,9 @@
      <!-- Name of a font family to use for body text. -->
     <string name="config_bodyFontFamily" translatable="false">arbutus-slab</string>
     <!-- Name of a font family to use for medium body text. -->
-    <string name="config_bodyFontFamilyMedium" translatable="false">arbutus-slab-medium</string>
+    <string name="config_bodyFontFamilyMedium" translatable="false">arbutus-slab-bold</string>
     <!-- Name of a font family to use for headlines. If empty, falls back to platform default -->
     <string name="config_headlineFontFamily" translatable="false">arbutus-slab</string>
     <!-- Name of the font family used for system surfaces where the font should use medium weight -->
-    <string name="config_headlineFontFamilyMedium" translatable="false">arbutus-slab-medium</string>
+    <string name="config_headlineFontFamilyMedium" translatable="false">arbutus-slab-bold</string>
 </resources>
-- 
2.17.1

